model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -356,7 +356,8 @@ def _parse_perf_throughput(self, section):
         Parses throughput from the perf analyzer output
         """
 
-        throughput = re.search('Throughput: (\d+\.\d+|\d+)', section)
+        throughput = re.search('Throughput: (\d+\.\d+e\+\d+|\d+\.\d+|\d+)',
```

Comment:
This assumes positive numbers.  Any chance this could every be negative?

Reply:
No, we can't have negative inferences per second ;)